pilot/lib/pilot/build_manager.rb

```diff
@@ -92,11 +92,7 @@ def distribute(options, build: nil)
         end
       end
 
-      if config[:notify_external_testers]
-        build.auto_notify_enabled = config[:notify_external_testers]
-      else
-        build.auto_notify_enabled = true
-      end
+      build.auto_notify_enabled = config[:notify_external_testers]
```

Comment:
This option is not optional.
https://github.com/fastlane/fastlane/blob/master/pilot/lib/pilot/options.rb#L96-L100